client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -749,6 +751,48 @@ export default function CampaignItemDetails( props: Props ) {
 							<div className="campaign-item-details__payment-container">
 								<div className="campaign-item-details__payment">
 									<div className="campaign-item-details__payment-row">
+										{ orders && orders.length > 0 && (
+											<div className="campaign-item-details__weekly-orders-row">
+												<div className="campaign-item-details__weekly-label"></div>
+												<div className="campaign-item-details__weekly-duration">
+													<span className="campaign-item-details__label">
+														{ translate( 'Duration' ) }
+													</span>
+												</div>
+												<div className="campaign-item-details__weekly-amount">
+													<span className="campaign-item-details__label">
+														{ translate( 'Amount' ) }
+													</span>
+												</div>
+											</div>
+										) }
+										{ orders && orders.length > 0
+											? orders.map( ( order: Order, index: number ) => {
+													const { total, created_at } = order;
+													const formatDuration = ( created_at: string ) => {
+														const originalDate = moment( created_at );
+														const weekBefore = originalDate.clone().subtract( 7, 'days' );
+
+														return `${ weekBefore.format( 'MMM, D' ) } - ${ originalDate.format(
```

Comment:
I've opted to use a non-US-centric date format here, instead of following the design.